app/scripts/modules/kubernetes/v2/serverGroup/details/resize/resize.html

```diff
@@ -0,0 +1,54 @@
+<div modal-page class="confirmation-modal">
+  <task-monitor monitor="ctrl.taskMonitor"></task-monitor>
+  <form role="form" name="resizeForm">
+    <modal-close dismiss="$dismiss()"></modal-close>
+    <div class="modal-header">
+      <h3>Resize {{ctrl.serverGroup.kind}} {{ctrl.serverGroup.displayName}}</h3>
+    </div>
+    <div class="modal-body confirmation-modal">
+      <div class="form-horizontal">
+        <div class="form-group form-inline">
+          <div class="col-md-3 sm-label-right">
+            Current
+          </div>
+          <div class="col-md-3">
+            <div class="input-group">
+              <input type="number"
+                     class="form-control input-sm highlight-pristine"
+                     ng-model="ctrl.current.desired"
+                     readonly/>
+              <span class="input-group-addon">pod<span ng-if="ctrl.current.desired != 1">s</span></span>
+            </div>
+          </div>
+        </div>
+        <div class="form-group form-inline">
+          <div class="col-md-3 sm-label-right">
+            Desired
+          </div>
+          <div class="col-md-3">
+            <div class="input-group">
+              <input type="number"
+                     class="form-control input-sm highlight-pristine"
+                     ng-model="ctrl.command.capacity.desired"
+                     min="0"
+                     required/>
+              <span class="input-group-addon">pod<span ng-if="ctrl.command.capacity.desired != 1">s</span></span>
+            </div>
+          </div>
+        </div>
+      </div>
+      <task-reason command="ctrl.command"></task-reason>
+    </div>
+    <div class="modal-footer">
+      <user-verification account="ctrl.serverGroup.account" verification="ctrl.verification"></user-verification>
+      <button type="submit" ng-click="ctrl.resize()" style="display:none"></button> <!-- Allows form submission via enter keypress-->
```

Comment:
I know this is everywhere...but can't you put `ng-submit="ctrl.resize()"` as a `form` attribute? Maybe I'm wrong.

Reply:
Sure thing - why do you prefer `ng-submit`?

Reply:
The hidden button feels a little hacky. Or not. Who knows.

Reply:
Not sure why, but the `ng-submit` form attribute causes the modal to close (but still submit the task) when you hit `enter`. With the hacky button you at get to see the task monitor do its thing.

Reply:
Oh. That's probably why it's being used. Now I know.